packages/fast-components-react-msft/app/app.tsx

```diff
@@ -100,12 +102,24 @@ export default class App extends React.Component<{}, IAppState> {
                     {componentFactory(examples, {...this.generateDesignSystem()})}
                 </SiteCategory>
                 <div slot={ShellSlot.infoBar}>
-                    <ColorPicker
-                        foregroundColor={this.state.foregroundColor}
-                        backgroundColor={this.state.backgroundColor}
-                        accentColor={this.state.accentColor}
-                        onColorUpdate={this.handleColorUpdate}
-                    />
+                    <span style={{display: "flex", alignItems: "center", height: "100%"}}>
+                        <label style={{marginRight: "8px"}}>density</label>
```

Comment:
do we want this to be lowercase "d"? Also, any reason we aren't using the Label component here?

Reply:
switched to div and Label

Reply:
I used a lowercase "d" because the theme colors are lowercase.